kustomize/stage/pod/chaos/pod-init-container-failed.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: kwok.x-k8s.io/v1alpha1
+kind: Stage
+metadata:
+  name: pod-init-container-running-failed
```

Comment:
Rename the file name to match this one.

IMO, one should distinguish between creating and running failed.

Reply:
Renamed the file to `pod-init-container-running-failed.yaml`.